05/Weather.js

```diff
@@ -12,13 +13,10 @@ class Weather extends React.Component {
 
   componentDidMount() {
     const { lat, lng } = this.props;
-    const apiKey = '338f265a980b48e19459c7cf7fb34484';
-    const apiUrl = `https://api.weatherbit.io/v2.0/current?key=${apiKey}&lat=${lat}&lon=${lng}&lang=pl`;
 
-    fetch(apiUrl)
-      .then((resp) => resp.json())
+    fetchWeatherData(lat, lng)
       .then((data) => this.setState({ data }))
-      .catch((err) => console.error('Error fetching ata: ', err));
+      .catch((err) => console.error('Error fetching data: ', err));
```

Comment:
Pamiętaj, jak robisz `.catch` przy `fetch` to tutaj nie będzie ten błąd już przechwycony - jedynie ten z this.setState() - tylko tak informacyjnie ;)